src/EnergyPlus/WeatherManager.cc

```diff
@@ -5782,7 +5779,6 @@ namespace Weather {
         static constexpr std::string_view routineName = "GetDesignDayData";
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        std::string units;
```

Comment:
[src/EnergyPlus/WeatherManager.cc:5785]:(style),[variableScope],The scope of the variable 'units' can be reduced.